### DISCUSSION TITLE
Domain And DNS

### DISCUSSION THREAD

**Discussion by soloappsnmobile** [vercel/community#2999] [Help] [answered]
### Summary

Vercel subdomain isn't secured. Why?

### Steps to Reproduce

My Vercel subdomain isn't secured. Why?

**Comment by amyegan** [comment]
If you explain a little more about what problem you're seeing, and share the domain, I can look into it for you

**Comment by soloappsnmobile** [comment]
Hi, Thanks for reaching out.
I purchased a domain [scanboxservices.com](url)
I then created a subdomain to be used in another service.  [scan.scanboxservices.com](url)
But for some reason, the subdomain isn't secured. I get [http ](url)instead of [https](url)

**Reply by smaeda-ks** [reply, marked as answer]
@soloappsnmobile 
Please read their documentation:
https://support.qr-code-generator.com/hc/en-us/articles/7665669262989-How-do-I-set-up-my-own-domain-

> *SSL certificates are only available with our Enterprise Plans.
> 
> White label short URLs do not have an SSL certificate by default. If you wish to secure your white label short URL with an SSL certificate (e.g. https://qr.yourdomain.com/), we can arrange one for you by request.*

This isn't a Vercel issue, as you're pointing the domain to a third-party provider on which the certificate should be installed. Our auto-certificate (Let's Encrypt) is only available if the domain is used in your Vercel project and serving traffic from there.

**Reply by mcbryan1** [reply]
Thank you very much

**Comment by soloappsnmobile** [comment]
![Screenshot (26)](https://github.com/vercel/community/assets/125050395/e035fd76-0757-4168-a98f-b1125733fdd6)


This is how I set up the subdomain
